packages/js/src/workouts/components/ConfigurationWorkout.js

```diff
@@ -263,11 +263,13 @@ export function ConfigurationWorkout( { toggleStep, toggleWorkout, isStepFinishe
 		if ( isStepFinished( "configuration", steps.enableTracking ) ) {
 			toggleStepEnableTracking();
 		} else {
-			updateTracking()
-				.then( toggleStepEnableTracking )
-				.catch( ( e ) => {
-					console.error( e );
-				} );
+			if ( ( state.tracking === 0 || state.tracking === 1 ) ) {
```

Comment:
Not sure why we check this, shouldn't it be already checked by line 173?

Reply:
Correct!